alacritty_terminal/src/term/mod.rs

```diff
@@ -308,7 +302,61 @@ pub struct Term<T> {
     damage: TermDamageState,
 
     /// Config directly for the terminal.
-    config: Terminal,
+    options: TermOptions,
+}
+
+/// Configuration options for the [`Term`].
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct TermOptions {
+    /// The amount of scrolling history to allocate.
+    pub scrolling_history: usize,
+
+    /// Defalut cursor style to reset cursor to.
+    pub default_cursor_style: CursorStyle,
+
+    /// Cursor style for Vi mode.
+    pub vi_mode_cursor_style: Option<CursorStyle>,
+
+    /// The characters which stop selection.
```

Comment:
This is misleading. These characters don't stop selection, they terminate semantic selection.